install_config/master_node_configuration.adoc

```diff
@@ -1203,8 +1203,21 @@ openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/openp
 
 The advanced audit feature provides several improvements over the
 xref:master-node-config-audit-config[basic audit functionality], including
-fine-grained events filtering and multiple output back ends. The following table
-contains additional options you can use.
+fine-grained events filtering and multiple output back ends. 
+
+To setup the advanced audit feature please add the following entries at the `openshift_master_audit_config` variable
+
+----
+openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/oscp-audit/-oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/security/adv-audit.yaml","logFormat":"json"}
```

Comment:
How do you pick values for these parameters?
Does /var/log/oscp-audit/-oscp-audit.log need to exist before you run this command?
Are there any limits to the "maximumFileRetentionDays" and "maximumFileSizeMegabytes" parameters?

Reply:
- The directory was just a example. Any suggestion where it should be as this file isn't handled yet by the installer him self.
- That's the same parameters as in the basic audit setup. The limits are mostly from the definition of the types.

https://github.com/openshift/origin/blob/8446a5c4d89862f96b64f13d9c4db0d30a210200/pkg/cmd/server/apis/config/types.go#L460-L465
```
	// Maximum number of days to retain old log files based on the timestamp encoded in their filename.
	MaximumFileRetentionDays int
	// Maximum number of old log files to retain.
	MaximumRetainedFiles int
	// Maximum size in megabytes of the log file before it gets rotated. Defaults to 100MB.
        MaximumFileSizeMegabytes int
```

As openshift uses lumberjack and I haven't seen there any limits documented https://github.com/natefinch/lumberjack/blob/v2.1/lumberjack.go#L79